index.bs

```diff
@@ -581,7 +587,7 @@ run these steps:
 2. Let |container| be the <a>containing block</a> of the <a>target</a>.
 3. While |container| is not the <a>intersection root</a>:
 	1. Map |intersectionRect| to the coordinate space of |container|.
-	2. If |container| has overflow clipping or a css <a>clip-path</a> property,
+	2. If |container| has an <a>overflow clip</a> or a CSS <A>clip-path</a> property,
```

Comment:
The casing change at the left of `clip-path` seems unintentional.

Reply:
Fixed.